workday/13.0/modules/ROOT/pages/workday-connector-design-center.adoc

```diff
@@ -1,66 +1,57 @@
-= Workday - Use Flow Designer to Configure Workday Connector 13.0 - Mule 4
-connectors:workday/workday-connector-design-center.adoc
+= Use Flow Designer to Configure Workday Connector 13.0 - Mule 4
+:page-aliases: connectors::workday/workday-design-center.adoc, connectors::workday/workday-connector-design-center.adoc
 
 Anypoint Design Center's Flow Designer enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
 
 To create an app in Flow Designer:
 
-* Configure the input source (trigger) for your app.
+* Configure the trigger (source) for your app.
 * Configure the target component for your app.
 
-For information on Design Center field values, see
+For information on connector field values, see
 the xref:workday-reference.adoc[Workday Connector Reference].
 
-== Configure the Input Source Trigger
+== Configure the Trigger
 
-A trigger starts your app. Workday Connector provides the Modified Object Trigger operation that starts your app when an object is created, modified, or deleted in Workday. Alternatively, you can use the HTTP Listener operation to start your app from a browser
-or with a command such as `curl`. Another alternative is to use Scheduler to start your app on a timed schedule.
+A trigger starts your app. Workday Connector provides the *On Modified Objects* trigger that starts your app when an object is created, modified, or deleted in Workday. Alternatively, you can use:
```

Comment:
Not sure we need the first sentence...:
```suggestion
Workday Connector provides the *On Modified Objects* trigger, which starts your app when an object is created, modified, or deleted in Workday. Alternatively, you can use:
```
OR this, which matches the following bullet-list items in using "to start"
```suggestion
Workday Connector provides the *On Modified Objects* trigger to start your app when an object is created, modified, or deleted in Workday. Alternatively, you can use:
```